features/systemspage.feature

```diff
@@ -14,7 +14,6 @@ Feature: Explore the main landing page
 
   Scenario: Completeness of the side navigation bar and the content frame
     Then I should see a "System Overview" text
-    And I should see a "No systems." text
```

Comment:
Instead of removing the step, which is actually checking that a "No systems." message exist when there is no systems, why don't modify the step to make it idempotent?

For example, something like: 
```
And I should see a "No systems." or "mgr.suse.de" text
```

That way, we're not loosing power but increasing it! 😄 

What do you think?